test/add-channel.test.js

```diff
@@ -1,20 +1,21 @@
-const test = require('ava');
-const nock = require('nock');
-const {stub} = require('sinon');
-const proxyquire = require('proxyquire');
-const {authenticate} = require('./helpers/mock-github');
-const rateLimit = require('./helpers/rate-limit');
+import test from 'ava';
+import nock from 'nock';
+import sinon from 'sinon';
+import quibble from "quibble"
```

Comment:
quotes are inconsistent with other imports

Reply:
it's one of the files where `xo` fails entirely due to 

> Parsing error: Cannot use keyword await outside an async function

It's really annoying, there is no way to configure `xo` to ignore this error. I'd say we look past formatting like this for now. We can run `prettier` across all files once we do the code changes

Reply:
works for me